crates/evm/src/executor/fork/database.rs

```diff
@@ -285,7 +286,9 @@ mod tests {
         let backend = SharedBackend::spawn_backend(Arc::new(provider), db.clone(), None).await;
 
         let mut db = ForkedDatabase::new(backend, db);
-        let address = B160::random();
+        let address = Address::from_word(b256!(
+            "000000000000000000000000d8da6bf26964af9d7eed9e03e53415d37aa96045"
+        ));
```

Comment:
FYI we have [`B*::random`](https://docs.rs/alloy-primitives/latest/alloy_primitives/struct.Address.html#method.random) but it's behind the `getrandom` feature flag, and also there's an [`address!`](https://docs.rs/alloy-primitives/latest/alloy_primitives/macro.address.html) macro

Reply:
sweet nice—will change to use that! rn i'm in the process of just making the PR compile without the migration itself (which is going to be stacked, per crate ideally)